gateway/node.go

```diff
@@ -225,6 +241,14 @@ func (gw *Node) checkTipsetHeight(ts *types.TipSet, h abi.ChainEpoch) error {
 	if h > ts.Height() {
```

Comment:
Is this also a misprint?

Reply:
This is not, I checked myself, and the `h` is not the head, but the tipset being queried, so I think that one works (I actually haven't changed that one).

BTW, feel free to merge if LGTY.